my-app/src/SearchPage.tsx

```diff
@@ -13,50 +13,57 @@ export interface SearchResult {
 }
 
 function SearchPage() {
-  const [searchQuery, setSearchQuery] = useState<string>('');
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [cardNumber, setCardNumber] = useState<string>('');
   const [searchResults, setSearchResults] = useState<SearchResult[]>([]);
 
-
   const submitForm = async (): Promise<void> => {
     try {
-      const response = await fetch('https://api.afg.com/account-search', {
+      const response = await fetch('https://api.afg.com/cardholders/search', {
         method: 'POST',
         headers: {
           'Content-Type': 'application/json',
         },
-        body: JSON.stringify(searchQuery),
+        body: JSON.stringify({ firstName, lastName, cardNumber }),
```

Comment:
Cardholders should expect card number token, not raw card number. We should probably have a tokenizer service for the raw card number.

Reply:
Also normalize, sanitize the value before sending it the BE